requirements.yml

```diff
@@ -10,7 +10,7 @@
   version: 1.1.0
 
 - name: openmicroscopy.network
-  version: 1.1.1
+  version: 1.1.0
```

Comment:
This version number decrement breaks the execution of `web-proxy` playbook. The changes proposed in https://github.com/openmicroscopy/ansible-role-network/pull/3 and released in `1.1.1` were required to fix the playbook execution with the current inventory variables. 

The issue that this tries to address was reported as https://github.com/openmicroscopy/ansible-role-network/issues/4.  Can we follow-up on https://github.com/openmicroscopy/ansible-role-network/issues/4#issuecomment-374883606 to decide whether 1- we can get the role fixed for all usages, 2- and/or there is a need to unify our internal variables definition.

Reply:
@sbesson - this change is no longer in the diff on this PR, so this is addressed by https://github.com/openmicroscopy/prod-playbooks/pull/62/commits/fa7838380e95abce056c36a3947bb8d56c0860b4